dask_planner/src/parser.rs

```diff
@@ -1222,17 +1240,23 @@ impl<'a> DaskParser<'a> {
         };
 
         Ok(DaskStatement::ShowSchemas(Box::new(ShowSchemas {
-            like: like_val,
+            catalog_name,
+            like,
         })))
     }
 
     /// Parse Dask-SQL SHOW TABLES [FROM] statement
     fn parse_show_tables(&mut self) -> Result<DaskStatement, ParserError> {
+        let mut catalog_name = None;
         let mut schema_name = None;
         if !self.parser.consume_token(&Token::EOF) {
```

Comment:
I know this was an existing code block that you only modified the inner behavior of but while we are at working on it I think it would benefit us to change this to an `if let` statement which would remove the requirement for the mutable `catalog_name` and `schema_name` variables.